src/shared/pages/user/addTeamToUser/__tests__/addTeamToUser.spec.tsx

```diff
@@ -169,26 +178,36 @@ describe('when the submit button is clicked', () => {
         });
     });
 
-    it('should call the service and dispach actions for the selected options', async () => {
-        expect.assertions(3);
-
-        act(() => {
-            const submitButton = getByText(wrapper.container, 'Add selected teams');
-            fireEvent.click(submitButton);
-        });
-
-        await wait(async () => {
-            expect(addUserToTeamSpy).nthCalledWith(1, {
-                label: '__user__',
-                value: '__userId__'
-            }, '__teamId1__');
-            expect(addUserToTeamSpy).nthCalledWith(2, {
-                label: '__user__',
-                value: '__userId__'
-            }, '__teamId2__');
-            expect(clearErrorsSpy).toBeCalled();
-        });
-    });
+    // it('should call the service and dispach actions for the selected options', async () => {
```

Comment:
Is this meant to be commented out? Or should this just be removed from the commit entirely?

Reply:
Ooops, no. Fixed.